hilti/runtime/include/types/integer.h

```diff
@@ -146,7 +148,7 @@ inline Bytes pack(integer::safe<T> i, ByteOrder fmt) {
         case ByteOrder::Undef: throw RuntimeError("attempt to pack value with undefined byte order");
     }
 
-    return Bytes(reinterpret_cast<Bytes::Base::value_type*>(raw), sizeof(raw));
+    return Bytes(std::move(str));
```

Comment:
If you construct a `Bytes`:

```suggestion
    return str;
```